addons/edi_product/models/edi_product_record.py

```diff
@@ -13,6 +13,15 @@ class EdiDocument(models.Model):
         "edi.inactive.product.record", "doc_id", string="Inactive Products"
     )
 
+    @api.depends("product_ids", "state")
```

Comment:
How often does this method get called e.g. when processing an item master? One call for each row on the input CSV? It could get expensive for large CSVs.

Reply:
Did some profiling, it gets called 4 times altogether when processing a file (no matter how large). Should be good from performance point of view.